src/main/java/oopprj/tetrisgame/tetris/TetrisLauncher.java

```diff
@@ -14,7 +15,8 @@ public void start(Stage stage) throws Exception {
         stage.setHeight(heightScreen);
 
         new ViewController(stage);
-        ViewController.getMainView();
+
+        ViewController.getChooseLevelView();
```

Comment:
you should hold the ViewController.getMainView() code. This is because the game will only run the main view. Choose level view will be display when user hit the new game button

Reply:
I got it, it just for test. New game button will be created in the future

Reply:
If it is used for testing, you should back it to default. Image that when I merge your code, Tetrislauncher.java in github (remote) will diffirent with all of local repos. It will create a conflict to other partner and they have to solve your error.